docs/hub/model-cards.md

```diff
@@ -89,8 +117,21 @@ This metadata will be used to display the base model on the model page. Users ca
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/base-model-ui-dark.png"/>
 </div>
 
+### Specifying a dataset
 
+You can specify the datasets used to train your model in the model card metadata section. The datasets will be displayed on the model page and users will be able to filter models by dataset. You should use the Hub dataset identifier, which is the same as the dataset's repo name as the identifier:
 
+```yaml
+datasets:
+- imdb
+- HuggingFaceH4/no_robots
+```
+
+### Specifying a task (`pipeline_tag`)
+
+You can specify the `pipeline_tag` in the model card metadata. The `pipeline_tag` indicates the type of task the model is intended for. This tag will be displayed on the model page and users can filter models on the Hub by task. This tag is also used to determine which [widget](./models-widgets.md#enabling-a-widget) to use for the model. 
+
+For `transformers` models, the pipeline tag is automatically inferred from the model's `config.json` file but you can override it in the model card metadata if required. Editing this field in the metadata UI will ensure that the pipeline tag is valid. Some other libraries with Hub integration will also automatically add the pipeline tag to the model card metadata.
```

Comment:
Some of this is duplicated with https://moon-ci-docs.huggingface.co/docs/hub/pr_1003/en/models-widgets#enabling-a-widget. Let's unify these. Probably we can move some of that over here, including the gist